pink_annotation_schema.ttl

```diff
@@ -51,7 +51,7 @@
 #################################################################
 
 ###  http://data.europa.eu/r5r/status
-dcatap:status rdf:type owl:AnnotationProperty .
+#dcatap:status rdf:type owl:AnnotationProperty .
```

Comment:
These properties are worse. They are added automatically by protege, unless we define them ourselves. 

I think that the correct solution is that we generate a ttl file from the default tripper keywords and import this ttl file.